app.rb

```diff
@@ -0,0 +1,120 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'rental'
+
```

Comment:
The issue that is mentioned by the previous reviewer is still prevalent.

![image](https://github.com/Ochiengsteven/school-library/assets/38088818/bb63e0cb-6515-437d-a18a-9361a3cf24c1)

This is due to the lack of `require_relative 'teacher'` statement in the `app.rb` file. Kindly ensure that the file is required so that Ruby has access to the teacher class.

Additionally, there's an `ArgumentError` issue with the teacher classs similar to the student class. Kindly pay attention to the flow of arguments that are passed to each class.

![image](https://github.com/Ochiengsteven/school-library/assets/38088818/8a8e800a-d776-4bc2-8712-6a8da7d34645)